_data-prepper/pipelines/configuration/processors/otel-metrics-raw.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: default
+title: otel metrics raw
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# otel metrics raw
+
+The `otel metrics raw` processor serializes a collection of `ExportMetricsServiceRequest` sent from the [otel-metrics-source]({{site.url}}{{site.baseurl}}//data-prepper/pipelines/configuration/sources/otel-metrics-source/) into a collection of string records.
+
+## Usage
+
+See the following `.yaml` configuration file:
+
+```
+processor:
+    - otel_metrics_raw_processor
+```
+
+## Configuration
+
+You can create detailed representations of histogram buckets and their boundaries. You can control this feature by using the following parameters:
+
+```yaml
+  processor:
+    - otel_metrics_raw_processor:
+        calculate_histogram_buckets: true
+        calculate_exponential_histogram_buckets: true
+        exponential_histogram_max_allowed_scale: 10
+        flatten_attributes: false
+```
+
+The following table describes the different parameters used with histogram buckets and their default values.
+
+| Parameter | Default value | Description |
```

Comment:
This table is incomplete and needs more information. What are the defaults if a parameter is provided? What do each of these parameters do/represent?

Reply:
Thanks for pointing it out! The table looks off. Let me check with team. We may need to update the code and the README in Data Prepper first.

Reply:
Checked again. Data prepper code is good. The table should be something like this:

| Parameter | Default value | Description |
| :---    | :---    | :---    |
| `calculate_histogram_buckets` | True | Whether to calculate histogram buckets or not |
| `calculate_exponential_histogram_buckets` | True | Whether to calculate exponential histogram buckets or not |
| `exponential_histogram_max_allowed_scale` | 10 | Max scale allowed in exponential histogram calculation |
| `flatten_attributes` | False | Whether to flatten the "attributes" field in the JSON data or not |